modules/network-observability-RTT-overview.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-RTT-overview_{context}"]
+= Round trip time tracing
```

Comment:
```suggestion
= Round-trip time tracing
```
@skrthomas  I see the hyphenated version for RTT on a search, but don't know if 'Round trip tracing' is the correct way for NO content? Otherwise LGTM!